### PR TITLE
add document about vlan configuration

### PR DESCRIPTION
**Pull request by penguhyang**
add vlan configuration document

### CODE REVIEW

File: docs/source/advanced/networks/vlan/vlan.rst [hunk 1]
```diff
@@ -1,2 +1,224 @@
...
+* chvlan
+* lsvlan
+* rmvlan
+
+These VLAN functions are supported for stand-alone nodes as well as virtual machines such as KVMs.
+
+Install the package
+-------------------
+
+The xCAT-vlan package is shipped with xcat-core in xCAT 2.9.1 and later. You can install it using the package installation tool for the os.
```
Comment (whowutwut) [inline docs/source/advanced/networks/vlan/vlan.rst]: Here is a situation where we do not need to specify the version anymore.  The documentation is now for 2.11 and 2.12..  

~~The xCAT-vlan package is shipped with xcat-core in xCAT 2.9.1 and later. You can install it~~ Install the ``xCAT-vlan`` package using the package ~~installation tool~~ manager ~~for the os.~~ on the OS.

File: docs/source/advanced/networks/vlan/vlan.rst [hunk 2]
```diff
@@ -1,2 +1,224 @@
...
+The xCAT-vlan package is shipped with xcat-core in xCAT 2.9.1 and later. You can install it using the package installation tool for the os. 
+
+For RHEL: ::
+
+    yum install xCAT-vlan
+
+For SLES: ::
+
+    zypper install xCAT-vlan
+
```
Comment (whowutwut) [inline docs/source/advanced/networks/vlan/vlan.rst]: I would suggest for the above, writing like... 

[RHEL]:  ``yum install xCAT-vlan``
[SLES]:  ``zypper install xCAT-vlan``
[Ubuntu]: ``apt-get install xCAT-vlan``

This keeps the document short and sweet

**Comment by whowutwut** [comment]
@cxhong  do you have any experience with VLAN to review these docs?

**Comment by cxhong** [comment]
The Doc looks good, but if the vlan configuration can't support our current OS (Ubuntu, rhel7 and sles12), should we wait for to merge this doc to github until we can support the current OS?

**Comment by cxhong** [comment]
Agree to merge, we will update the Doc after we can support the newest OS.

**Pull request merged**